Author: Thomas Gaugler <thomas@dadie.net>
Description: Disable name mangling for the DllMain function so that the Math plugin could be properly initialized. The issue was reported upstream: http://sf.net/support/tracker.php?aid=2835731

--- nsis-2.45-src/Contrib/Math/Source/Math.c    2009-02-04 15:08:31.000000000 +0100
+++ nsis-2.45/Contrib/Math/Source/Math.c        2009-08-11 09:17:01.000000000 +0200
@@ -1552,7 +1552,7 @@
   }
 }
 
-BOOL WINAPI DllMain(HINSTANCE hInst, ULONG ul_reason_for_call, LPVOID lpReserved)
+extern "C" BOOL WINAPI DllMain(HINSTANCE hInst, ULONG ul_reason_for_call, LPVOID lpReserved)
 {
     g_hInstance = hInst;
     CleanAll(ul_reason_for_call == DLL_PROCESS_ATTACH);
